rllib/offline/offline_prelearner.py

```diff
@@ -288,7 +298,7 @@ def convert(sample, space):
                             unpack_if_needed(batch[schema[Columns.NEXT_OBS]][i]),
                             observation_space,
                         )
-                        if Columns.NEXT_OBS in input_compress_columns
+                        if Columns.OBS in input_compress_columns
```

Comment:
Just to be 100% safe, should we even allow NEXT_OBS to be present in `input_compress_columns`, then? If we ignore it anyways.

Reply:
Great question. Right now we "assume" only that a user will never put `Columns.NEXT_OBS` into `input_compress_columns` AND and the same time NOT `Columns.OBS`, but this could of course be different.

My idea is also to ONLY ALLOW `Columns` constants in `input_compress_columns` b/c if different column names are used, the user has to nevertheless use a custom `input_read_schema` that maps `Columns` constants to her custom column names.